collection-spec/json-schema/collection.json

```diff
@@ -94,6 +114,27 @@
           "type": "string"
         }
       }
+    },
+    "aggregation": {
+      "type": "object",
+      "required": ["agg_type"],
+      "properties": {
+        "agg_type": {
+          "title": "Type of aggegration",
+          "description": "Type of aggegation operation to apply: join_new, join_existing, union",
+          "type": "string"
+        },
+        "attribute_name": {
+          "title": "Name of attribute",
+          "description": "Name of attribute across which to aggregate.",
+          "type": "string"
+        },
+        "options": {
+          "title": "Options",
+          "description": "Optional aggegration settings.",
```

Comment:
Maybe we should be a little more detailed here. What kind of options? Do we want to explicitly reference xarray? In our implementation, these options will be passed directly to xarray.concat and xarray,merge.